kubernetes/resource_kubernetes_service_v1_test.go

```diff
@@ -819,6 +819,36 @@ func TestAccKubernetesServiceV1_ipFamilies(t *testing.T) {
 	})
 }
 
+func TestAccKubernetesServiceV1_trafficDistribution(t *testing.T) {
+	var conf corev1.Service
+	name := acctest.RandomWithPrefix("tf-acc-test")
+	resourceName := "kubernetes_service_v1.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
```

Comment:
Please add here `skipIfClusterVersionLessThan` to make sure these test runs on Kubernetes 1.31+ only.